pom.xml

```diff
@@ -122,7 +122,8 @@
   </scm>
 
   <properties>
-    <mavenVersion>3.6.3</mavenVersion>
+    <mavenMinVersion>3.6.3</mavenMinVersion>
+    <mavenVersion>3.9.6</mavenVersion>
```

Comment:
I'm not sure if it good idea.
After it we will can use newer methods from API ... and we will have an exception in runtime on older Maven.

Reply:
I agree that this is wrong, but if we will run tests with 3.6.3 we should sooner or later catch it